app/controllers/bike_share_app.rb

```diff
@@ -1,3 +1,158 @@
+require_relative '../models/station'
+require_relative '../helpers/bike_share_app_helper'
+
 class BikeShareApp < Sinatra::Base
 
+  include BikeShareAppHelper
```

Comment:
do we need that method available throughout this entire controller? or could we make the `format_date` method like so:

```ruby
module BikeShareAppHelper
  def self.format_date(date)
     ...
  end
end
```

this way - we don't need to include the module explicitly and have it available throughout the entire controller, we can just use it when necessary. for example, on line 32 we could say:

```ruby
params[:station][:installation_date] = BikeShareAppHelper.format_date(params[:station][:installation_date])
```